context.go

```diff
@@ -445,6 +445,24 @@ func (ctx *Context) JSON(jsonObjectOrArray interface{}) error {
 	return ctx.WriteJSON(http.StatusOK, jsonObjectOrArray)
 }
 
+// ReadXML reads XML from request's body
+func (ctx Context) ReadXML(xmlObject interface{}) error {
```

Comment:
@wsantos in fact I don't think it will be necessary, because only in the case of failure to parse the request's body the "error" will be returned by function. But for code coverage statistics I agree with you.